rmgpy/species.py

```diff
@@ -443,9 +443,17 @@ def to_cantera(self, use_chemkin_identifier=False):
             else:
                 element_dict[symbol] += 1
         if use_chemkin_identifier:
-            ct_species = ct.Species(self.to_chemkin(), element_dict)
+            label = self.to_chemkin()
         else:
-            ct_species = ct.Species(self.label, element_dict)
+            label = self.label
+
+        if self.contains_surface_site() and element_dict["X"] > 1:
+            # for multidentate adsorbates, 'size' is the same as 'sites'? for some reason, cantera won't take the input 'size,' so will need to use 'sites'
```

Comment:
sites and size are accidentally switched in the second half of this comment
```suggestion
            # for multidentate adsorbates, 'size' is the same as 'sites'? for some reason, cantera won't take the input 'sites,' so will need to use 'size'
```